src/workerd/api/hyperdrive.c++

```diff
@@ -37,8 +37,9 @@ jsg::Ref<Socket> Hyperdrive::connect(jsg::Lock& js) {
   // TODO(someday): Support TLS? It's not at all necessary since we're connecting locally, but
   // some users may want it anyway.
   auto nullTlsStarter = kj::heap<kj::TlsStarterCallback>();
-  auto sock = setupSocket(js, kj::mv(conn), kj::none, kj::mv(nullTlsStarter),
-                  false, kj::str(this->randomHost), false);
+  auto sock = setupSocket(js, kj::mv(conn), kj::str(getHost(), ":", getPort()),
+                          kj::none, kj::mv(nullTlsStarter), false,
```

Comment:
@xortive do you think it's worth providing some sort of remote address for sockets that are connected to Hyperdrive? It's clearly not a pressing issue, but might eventually be nice.